src/elements/x-safe.js

```diff
@@ -69,11 +70,21 @@ export default class XSafe extends MixinRedux(XElement) {
             <section class="content nimiq-dark content-width">
                 <nav class="actions floating-actions">
                     <div class="floating-btn">
-                        <button new-tx><span>Send</span></button>
+                        <button new-tx>
+                            <div>
+                                <svg xmlns="http://www.w3.org/2000/svg"><path d="M15,20H9V12H4.16L12,4.16L19.84,12H15V20Z" /></svg>
```

Comment:
why as `svg` now?
Also the arrows in Julian's design are thinner, which looks better

Reply:
fill color can be applied that way.